maquetado/scss/fonts.scss

```diff
@@ -1,3 +1,5 @@
+$nav-height: 90px;
```

Comment:
This are not fonts, you should move them to `navbar.css`

Same with `title-decoration`, should be in `titles.scss`